docs/spicedb-dedicated/fgam.md

```diff
@@ -0,0 +1,141 @@
+
+### Policy
+
+ A policy defines the conditions under which a Principal (for now just the Service Account) is granted API access via a Role. A policy represents a 1:1 mapping Service Account <-> Role, although a Service Account may be granted access via different policies. The result of granting multiple roles via multiple policies to a Service Account is additive: if one policy grants permission and the other does not, the policy that granted API access wins.
+
+### Token
+
+A Service Account can have zero or more tokens, which represents a long-lived credential that identifies the Service Account. A client application would present the token as part of an API request, which grants the bearer the ability to talk to the APIs granted through the policy and role.
+
+## Access Management Interface
+
+The access management interface can be accessed in the navigation menu of your Permission System page
+
+![Service Account Page](/img/fgam/service-account-page.png)
+
+The Service Account Page allows you to:
+
+- see existing service accounts
+- see details of a specific service account
+- delete a service account
+
+Please note that Service Accounts cannot be deleted if referenced by a Policy
+
+The Roles page offers similar functionality, but for Roles:
+
+![Roles Page](/img/fgam/roles-page.png)
+
+- List existing roles
+- Navigate to the details page of a role
+- delete a role
+
+Please note that roles cannot be deleted if referenced by a Policy.
+
+The Policies page shows:
+
+- existing policies
+- navigate to the details page of a policy
+- delete a policy
+
+![Policies Page](/img/fgam/policies-page.png)
+
+## Advanced Roles with Conditions
+
+SpiceDB Dedicated Fine-Grained Access Management offers advanced permissions by letting users define dynamic conditions that need to be met at request time for the request to be authorized. This is achieved by augmenting fine-grained permissions with Google's CEL expression language.
+
+Each role permission provides an input text box to introduce an conditional expression, which is optional. When included, an API request will be authorized if:
+
+- the service account has been granted a permission
+- the expression of the permission is met
+
+![Advanced Role Conditions](/img/fgam/advanced-role-conditions.png)
+![advanced-role-conditions](https://user-images.githubusercontent.com/6774726/226452587-f8c3b9da-8e2d-4c13-861f-094b7719546e.png)
+
+SpiceDB Enterprise will expose the protobuf payload to the expression language, so the following variables will be available to be used in the expression:
+
+- `WriteRelationshipsRequest`
+- `ReadRelationshipsRequest`
+- `DeleteRelationshipsRequest`
+- `WriteSchemaRequest`
+- `ReadSchemaRequest`
+- `CheckPermissionRequest`
+- `LookupResourcesRequest`
+- `LookupSubjectsRequest`
+- `ExpandPermissionTreeRequest`
+- `WatchRequest`
```

Comment:
Because there can only be one request at a time, I suppose it doesn't matter if there are any name conflicts long as the correct type for your request is available.